### PR TITLE
EPMRPP-78795 || Adjust the rest plugin extensions to WMF

### PR DESCRIPTION
**Pull request by AmsterGet**

### CODE REVIEW

File: app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx [hunk 1]
```diff
@@ -142,13 +138,17 @@ export class AdminSidebar extends Component {
...
       },
     ];
-    const extensionItems = this.props.extensions.map((extension) => ({
-      onClick: this.handleClickButton(),
-      link: { type: PLUGIN_UI_EXTENSION_ADMIN_PAGE, payload: { pluginPage: extension.name } },
-      icon: extension.buttonIcon,
-      message: extension.buttonLabel || extension.name,
-    }));
-    return [...items, ...extensionItems];
+    if (extensions.length) {
```
Comment (Vadim73i) [inline app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx:141]: What is this check for?
forEach can operate on an empty array

File: app/src/layouts/appLayout/appSidebar/appSidebar.jsx [hunk 2]
```diff
@@ -150,8 +145,14 @@ export class AppSidebar extends Component {
       message: <FormattedMessage id={'Sidebar.settingsBnt'} defaultMessage={'Project settings'} />,
     });
 
-    if (extensionComponents.length) {
-      extensionComponents.forEach((item) => topItems.push({ ...item, onClick: onClickNavBtn }));
+    if (extensions.length) {
```
Comment (Vadim73i) [inline app/src/layouts/appLayout/appSidebar/appSidebar.jsx:148]: The same here

File: app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx (hunk 1 above)
Comment (AmsterGet) [reply app/src/layouts/adminLayout/adminSidebar/adminSidebar.jsx:141]: Fixed.

File: app/src/layouts/appLayout/appSidebar/appSidebar.jsx (hunk 2 above)
Comment (AmsterGet) [reply app/src/layouts/appLayout/appSidebar/appSidebar.jsx:148]: Fixed.

**Review by Vadim73i — APPROVED** [review]

**Review by chivekrodis — APPROVED** [review]

**Pull request merged**